nextflow_schema.json

```diff
@@ -103,30 +104,30 @@
                 },
                 "genome": {
                     "type": "string",
-                    "description": "Name of iGenomes reference.",
+                    "description": "Specify name of iGenomes reference.",
```

Comment:
These three genome/iGenomes ones are from teh tempalte, I would not change these

Nor anything between here to my next comment...